build.sh

```diff
@@ -0,0 +1,16 @@
+set -e
+export RUST_INSTALL_PATH=$PWD/rust
+mkdir -p $RUST_INSTALL_PATH
+export RUSTUP_HOME=$RUST_INSTALL_PATH/rustup
+export CARGO_HOME=$RUST_INSTALL_PATH/cargo
+export PATH=$CARGO_HOME/bin:$PATH
+export RB_SYS_DOCK_CACHE_DIR=$PWD/cache
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+rustup install stable
+rustup default stable
+mkdir -p /.cargo
+echo "[patch.crates-io.eppo_core]" > "/.cargo/config.toml"
+echo "path = '$PWD/eppo_core'" >> "/.cargo/config.toml"
```

Comment:
@rasendubi I added the patch here but this doesn't seem to be working with `rb-sys-dock`, do you have any bandwidth to take a look? that would be highly appreciated

Reply:
oh, that might be because `rb-sys-dock` runs another docker container for the actual build, so it does not copy cargo config. There should be a parameter for `rb-sys-dock` to run a command before the build